scripts/ci/verify_linter.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+set -ex
+
+# Install CLI & Dev Tools
+echo "Installing azure-cli-dev-tools and azure-cli..."
+pip install --pre azure-cli --extra-index-url https://azurecliprod.blob.core.windows.net/edge
```

Comment:
Print the az version so that we can check if the az is the latest build from the log.